mbs_results/validate_estimation.py

```diff
@@ -33,10 +60,17 @@ def validate_estimation(
                 """
             )
 
-    weighted_sum = (df[auxiliary] * df[design_weight] * df[calibration_factor]).sum()
     unweighted_sum = df[auxiliary].sum()
 
-    if weighted_sum != unweighted_sum:
+    sampled_df = df[df[sampled] == 1]
+
+    weighted_sum = (
+        sampled_df[auxiliary]
+        * sampled_df[design_weight]
+        * sampled_df[calibration_factor]
+    ).sum()
+
+    if not (isclose(weighted_sum, unweighted_sum)):
```

Comment:
Maybe add a comment to mention default tolerance? Not important, you can get from docs anyway